src/lib/y2storage/encryption_processes/swap.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/encryption_type"
+require "y2storage/encryption_processes/base"
+
+module Y2Storage
+  module EncryptionProcesses
+    # The enryption process that allows to create and identify a randomly
```

Comment:
Typo: enryption.

Reply:
I would re-word it a little: Encryption process that allows to encrypt a swap device by using plain encryption and random password.